src/version_info.rs

```diff
@@ -32,17 +31,10 @@ pub fn get_version_info(store: &dyn Storage) -> Result<VersionInfoV1, ContractEr
         Ok(_) => version_info_result,
         // version support added in 0.15.0, all previous versions used ContractInfo for version tracking
         // if VersionInfo doesn't exist, try ContractInfo
-        Err(_) => match get_legacy_contract_info(store) {
-            #[allow(deprecated)]
-            Ok(contract_info) => Ok(VersionInfoV1 {
-                definition: contract_info.definition,
-                version: contract_info.version,
-            }),
-            Err(_) => Err(ContractError::UnsupportedUpgrade {
-                source_version: "UNKNOWN".to_string(),
-                target_version: PACKAGE_VERSION.into(),
-            }),
-        },
+        Err(_) => Err(ContractError::UnsupportedUpgrade {
```

Comment:
I know this was here before, but does this error type make sense? It wouldn't really be an unsupported upgrade, it would probably be a bad storage state

Reply:
It doesn't make a ton of sense, but since this is in support of removing old migration code, it might make sense to remove it as well.